lib/vmdb/loggers.rb

```diff
@@ -49,7 +48,6 @@ def self.create_loggers
 
       $audit_log          = AuditLogger.new(path_dir.join("audit.log"))
       $container_log      = ContainerLogger.new
-      $journald_log       = create_journald_logger
```

Comment:
I think you can get away with just commenting this one line for now.  Maybe comment with a TODO.

Reply:
:+1: done